exercises/J-functions/exercise.js

```diff
@@ -1,7 +1,25 @@
 function halve(number) {
+  return number/2;
```

Comment:
The code is more readable if you put spaces between the operator and the operand, like you did on line 12.